task/prefetch-dependencies-oci-ta/0.2/prefetch-dependencies-oci-ta.yaml

```diff
@@ -372,6 +372,11 @@ spec:
 
         INPUT=$(inject_rpm_summary_flag "$INPUT")
 
+        if [[ "${INPUT}}" == *"gomod"* ]] && [[ "$(cd "/var/workdir/source" && ! git fetch --tags)" ]]; then
```

Comment:
why only for 'gomod' ?

Reply:
Only gomod is affected by the bug we're trying to solve here (no other package manager fetches tags, not at the moment at least).

Reply:
right, but it wouldn't hurt to do it always